coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/aggregation/GlobalAggregationCalculatorTest.kt

```diff
@@ -738,6 +731,18 @@ class GlobalAggregationCalculatorTest {
       val expectedAggregations: List<BlobsToAggregate>
     )
 
+    private fun assertOneToAggregationSizeUsesAggregationSize(
+      endSize:
+        UInt,
```

Comment:
Same issue with formatting. Sorry for not raising this in the same comment before

Reply:
sorted